src/index.html

```diff
@@ -17,6 +22,39 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo">
+        <img
+          src="images/logo.png"
+          alt="header"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              class="is-active"
+              href="apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li><a href="samsung">Samsung</a></li>
+          <li><a href="smartphones">Smartphones</a></li>
+          <li>
+            <a
+              data-qa="hover"
+              href="laptops"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li><a href="gadgets">Gadgets</a></li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading.